README.md

```diff
@@ -365,18 +373,27 @@ Another example will show how you can run and do the functional test of the serv
 
 ## Unit testing
 
-Write a unit test for the `calculate_ip_address` function. Start by creating a new folder `tests` inside a python folder of the loopback package. Then create a file inside called `loopback_test.py`. You will use Python built-in Unit test library to implement a unit test that will check if function works correctly and returns correct host address for a given prefix. Add the following code to the created file:
+Write a unit test for the `calculate_ip_address` function. Start by creating a new folder `tests` inside a python folder of the loopback package. Then create a file inside called `test_loopback.py`. 
+
+```
+mkdir ~/src/loopback/python/tests
+touch ~/src/loopback/python/tests/test_loopback.py
+touch ~/src/loopback/python/tests/__init__.py
+```
+
+You will use Python built-in Unit test library to implement a unit test that will check if function works correctly and returns correct host address for a given prefix. Add the following code to the created file:
```

Comment:
> _Add the following code to the created file:_

Can you mention the filename? can be something like: 

_Add the following code to the created  file`test_loopback.py_